<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="AASHYD-847B8Y2" failures="1" tests="1" name="com.advrt.testcases.AD_Asset_Admin_AuditTest" time="171.756" errors="0" timestamp="2024-03-13T10:30:12 IST" skipped="0">
  <testcase classname="com.advrt.testcases.AD_Asset_Admin_AuditTest" name="AD_Asset_Audit_018" time="171.756">
    <failure message="The following asserts failed:
	expected [Asset : &quot;Ast018&quot;  &quot;Description&quot; field modified from &quot;  Add to TestAdd &quot; by User Id : &quot;Kiranc1&quot; , User Name : &quot;Kiranc1&quot;.] but found [Asset : &quot;Ast018&quot;  &quot;Description&quot; field modified from &quot; Add&quot; to &quot;TestAdd &quot; by User Id : &quot;Kiranc1&quot; , User Name : &quot;Kiranc1&quot;.],
	expected [Asset : &quot;Ast018&quot;  &quot;Description&quot; field modified from &quot;  TestAdd to  &quot; by User Id : &quot;Kiranc1&quot; , User Name : &quot;Kiranc1&quot;.] but found [Asset : &quot;Ast018&quot;  &quot;Description&quot; field modified from &quot; TestAdd&quot; to &quot; &quot; by User Id : &quot;Kiranc1&quot; , User Name : &quot;Kiranc1&quot;.],
	expected [Asset : &quot;Ast018&quot;  &quot;Description&quot; field modified from &quot;   to NewComment &quot; by User Id : &quot;Kiranc1&quot; , User Name : &quot;Kiranc1&quot;.] but found [Asset : &quot;Ast018&quot;  &quot;Description&quot; field modified from &quot; &quot; to &quot;NewComment &quot; by User Id : &quot;Kiranc1&quot; , User Name : &quot;Kiranc1&quot;.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Asset : "Ast018"  "Description" field modified from "  Add to TestAdd " by User Id : "Kiranc1" , User Name : "Kiranc1".] but found [Asset : "Ast018"  "Description" field modified from " Add" to "TestAdd " by User Id : "Kiranc1" , User Name : "Kiranc1".],
expected [Asset : "Ast018"  "Description" field modified from "  TestAdd to  " by User Id : "Kiranc1" , User Name : "Kiranc1".] but found [Asset : "Ast018"  "Description" field modified from " TestAdd" to " " by User Id : "Kiranc1" , User Name : "Kiranc1".],
expected [Asset : "Ast018"  "Description" field modified from "   to NewComment " by User Id : "Kiranc1" , User Name : "Kiranc1".] but found [Asset : "Ast018"  "Description" field modified from " " to "NewComment " by User Id : "Kiranc1" , User Name : "Kiranc1".]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at com.advrt.testcases.AD_Asset_Admin_AuditTest.AD_Asset_Audit_018(AD_Asset_Admin_AuditTest.java:864)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- AD_Asset_Audit_018 -->
</testsuite> <!-- com.advrt.testcases.AD_Asset_Admin_AuditTest -->
